lib/vagrant-openshift/action/run_sti_tests.rb

```diff
@@ -52,7 +52,7 @@ def call(env)
           _,_,env[:test_exit_code] = sudo(env[:machine], %{
 set -e
 pushd #{Constants.build_dir}/source-to-image >/dev/null
-export PATH=/data/src/github.com/openshift/source-to-image/_output/local/go/bin:/data/src/github.com/openshift/source-to-image/_output/local/bin/linux/amd64:$PATH
+export PATH=/data/src/github.com/openshift/source-to-image/_output/local/go/bin:/data/src/github.com/openshift/source-to-image/_output/local/bin/linux/amd64:/data/src/github.com/openshift/source-to-image/_output/local/bin/windows/amd64:$PATH
```

Comment:
Wouldn't someone running these tests locally on their own machine need the `$PATH` augment as well? Why not have the `$PATH` change inside of the test script?